django_socio_grpc/utils/utils.py

```diff
@@ -22,20 +22,14 @@ async def async_generator():
                 async for item in response:
                     yield item
             except Exception as e:
-                if process_exceptions:
-                    await process_exceptions(e, request)
-                else:
-                    raise e
+                raise e
```

Comment:
keep it

Reply:
We changed the calls to not use this anymore though. I put it back but it's not used anymore.